libsawtooth/Cargo.toml

```diff
@@ -24,6 +24,7 @@ description = """\
 """
 
 [dependencies]
+base64 = { version = "0.12.3", optional = true }
```

Comment:
Unless we have a strong reason to, we generally only specify the major/minor version for dependencies. If we specify the patch version, that would prevent cargo from automatically pulling in any patches/fixes if there's a newer patch release (like 0.12.4).

Reply:
Fixed